baseline.yaml

```diff
@@ -649,6 +649,29 @@ criteria:
     security_insights_value: # TODO
     scorecard_probe: # TODO
 
+  - id: OSPS-DO-13
+    maturity_level: 1
+    category: Documentation
+    criteria: |
+       The PROJECT DOCUMENTATION MUST include a 
+       descriptive statement about how the project 
+       is supported/maintained and what is not supported. 
+       Projects are encouraged to provide "End of 
+       Support" statements when versions or features 
+       are deprecated or no longer supported.
+    objective: |
+       Identify and set expectations for downstream 
+       consumers around what updates the project may
+       or may not provide and when no further updates
+       will be provided.
+    implementation: |
+       Create a status check that checks the project's
+       version control system for support and/or lifecycle
+       statements.
```

Comment:
```suggestion
       Create a status check that checks the project's
       version control system for support statements, lifecycle
       statements, or related machine-readable metadata.
```

Reply:
great suggestion, I like both!

Reply:
I think this is a good thing to discuss with Security Insights in mind. Currently, the design of SI is to have one insights file per repo, and [lifecycle](https://github.com/ossf/security-insights-spec/blob/main/specification/project-lifecycle.md) is included as (IIRC) an optional value.

Obligatory note: we have yet to revise security insights.

Reply:
I don't see anything in my edit that would preclude the use of Security Insights, but I don't think we want to have Baseline be tightly coupled to it.